driverconfig/config.yaml

```diff
@@ -285,19 +285,19 @@ csiSideCars:
   - name: csi-metadata-retriever
     images:
       - version: v121
-        tag: dellemc/csi-metadata-retriever:v1.3.0
+        tag: dellemc/csi-metadata-retriever:v1.4.0
       - version: v122
-        tag: dellemc/csi-metadata-retriever:v1.3.0
+        tag: dellemc/csi-metadata-retriever:v1.4.0
```

Comment:
Why is this change required for K8s 1.22 and 1.23

Reply:
removed 1.22 and 1.23